explorer/main.cpp

```diff
@@ -43,14 +43,18 @@ auto ExplorerMain(int argc, char** argv, void* static_for_main_addr,
   llvm::errs().tie(&llvm::outs());
 
   cl::opt<std::string> input_file_name(cl::Positional, cl::desc("<input file>"),
-                                       cl::Required);
+                                       cl::Required, cl::value_desc("filename"),
+                                       cl::init("input.carbon"));
```

Comment:
```suggestion
                                       cl::Required);
```

This flag is positional and required. I don't think either value_desc or init have an effect, but if I'm missing something, can you explain with a diff in help output?